package.json

```
@@ -35,10 +35,10 @@
     "prepublishOnly": "./scripts/prepare-artifacts.sh --network $npm_config_network"
   },
   "dependencies": {
-    "@keep-network/keep-core": ">1.8.0-dev <1.8.0-pre",
-    "@keep-network/tbtc": ">1.1.2-dev <1.1.2-pre",
+    "@keep-network/keep-core": ">1.8.0-dev <1.8.0-goerli",
+    "@keep-network/tbtc": ">1.1.2-dev <1.1.2-goerli",
     "@thesis/solidity-contracts": "github:thesis/solidity-contracts#4985bcf",
-    "@threshold-network/solidity-contracts": ">1.1.0-dev <1.1.0-ropsten",
+    "@threshold-network/solidity-contracts": ">1.2.0-dev <1.2.0-goerli",
```

Could we use `development` tag?
```suggestion
    "@threshold-network/solidity-contracts": "development",
```

Same for other dependencies.

---

71f44ba3fcb99e3ccd576fc45d155d42566b9856